### PR TITLE
Feat/category table

### PR DESCRIPTION
**Pull request by GreygorySmirnov**
Add the the table for the categories.

### CODE REVIEW

**Review by WalldoBrie — APPROVED** [review]
Everything looks clean, good job.

**Comment by asm22ae** [comment]
Je pense que c est parafait!

**Review by Pannx — APPROVED** [review]
Superbe code !

File: src/components/molecules/myCategoryTable/myCategoryTable.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import DeleteForeverIcon from "@mui/icons-material/DeleteForever";
+import AddBoxIcon from "@mui/icons-material/AddBox";
+import { useRouter } from "next/navigation";
```
Comment (Pannx) [inline src/components/molecules/myCategoryTable/myCategoryTable.tsx:16]: Why not import it form next/router instead ? Why next/navigation ?

**Review by Pannx — APPROVED** [review]

**Pull request merged**